packages/iTwinUI-react/src/core/Dialog/DialogMain.tsx

```diff
@@ -117,13 +148,19 @@ export const DialogMain = React.forwardRef<HTMLDivElement, DialogMainProps>(
             'iui-dialog-default': styleType === 'default',
             'iui-dialog-full-page': styleType === 'fullPage',
             'iui-dialog-visible': isOpen,
+            'iui-dialog-draggable': isDraggable,
           },
           className,
         )}
         role='dialog'
         ref={refs}
         onKeyDown={handleKeyDown}
         tabIndex={-1}
+        style={{
+          transform,
```

Comment:
It looks like this is being applied to the non-draggable dialog too

Reply:
Added ternary